snuba/subscriptions/executor_consumer.py

```diff
@@ -247,6 +277,34 @@ def __init__(
             self.__metrics, "executor.concurrent.clickhouse"
         )
 
+    @property
+    def max_concurrent_queries(self) -> int:
+        return self.__max_concurrent_queries
+
+    def __calculate_max_concurrent_queries(
+        self,
+        partitions: Mapping[Partition, int],
+        partition_count: int,
```

Comment:
We are dealing with a lot of different partition counts here, `total_partition_count` would be clearer